<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainUIClass</class>
 <widget class="QMainWindow" name="mainUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1382</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="window_grid_layout">
    <item row="1" column="3">
     <widget class="MultiCompleterLineEdit" name="query_line_edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="query_push_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6" rowspan="5">
     <widget class="QGroupBox" name="tags_group_box">
      <property name="title">
       <string>Tags</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="new_tag_push_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Create Tag</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="delete_tag_push_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete tag</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableView" name="tag_table_view">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="tag_search_line_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="all_media_push_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>All media</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="tagless_media_push_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>NoTag Media</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="clear_media_push_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Clear media</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" rowspan="4" colspan="2">
     <widget class="MediaListView" name="media_list_view"/>
    </item>
    <item row="2" column="0" rowspan="4" colspan="3">
     <widget class="QGroupBox" name="media_info_group_box">
      <property name="title">
       <string>Media Info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="media_info_info_groupbox">
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="media_info_id_title_label">
            <property name="text">
             <string>ID:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="media_info_id_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="media_info_name_title_label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="media_subdir_title_label">
            <property name="text">
             <string>Subdir:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="media_info_name_line_edit"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="media_info_subdir_line_edit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="media_hash_title_label">
            <property name="text">
             <string>Hash:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="media_info_hash_line_edit"/>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="media_info_hash_regen_push_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Regenerate Hash</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="MultiCompleterLineEdit" name="media_tag_search_line_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="add_media_tag_push_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QListView" name="media_tag_list_view"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="media_info_thumb_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="log_docked_widget">
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListView" name="log_list_view">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MediaListView</class>
   <extends>QListView</extends>
   <header>medialistview.h</header>
  </customwidget>
  <customwidget>
   <class>MultiCompleterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>multicompleterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainUI.qrc"/>
 </resources>
 <connections/>
</ui>
